.github/dependabot.yml

```diff
@@ -20,20 +20,56 @@ updates:
       interval: monthly
     commit-message:
       prefix: meta
+    ignore:
+      - dependency-name: '@node-core/*'
     cooldown:
       default-days: 3
     groups:
-      format:
+      orama:
         patterns:
-          - 'prettier'
+          - '@orama/*'
+      cli:
+        patterns:
+          - 'commander'
+          - '@clack/prompts'
       lint:
         patterns:
+          - 'prettier'
           - 'eslint'
           - 'eslint-*'
-          - '@eslint/*'
+          - 'lint-staged'
           - 'globals'
-          - 'stylelint-*'
-      orama:
+          - '@eslint/*'
+      unist:
         patterns:
-          - '@orama/*'
+          - 'unified'
+          - 'unist-*'
+          - 'vfile'
+      remark:
+        patterns:
+          - 'remark-*'
+          - 'shiki'
+      rehype:
+        patterns:
+          - 'rehype-*'
+      ast:
+        patterns:
+          - 'estree-*'
+          - 'hast-*'
+          - 'mdast-*'
+          - 'hastscript'
+          - 'acorn'
+      recma:
+        patterns:
+          - 'recma-*'
+      compiling:
+        patterns:
+          - '@minify-html/node'
+          - '@rollup/*'
+          - 'rolldown'
+          - 'lightningcss'
+      react:
+        patterns:
+          - 'preact'
+          - 'preact-*'
     open-pull-requests-limit: 10
```

Comment:
You're going to want to bump this, given you've defined 10 groups already, excluding any dependencies that aren't in a group